[CTOR-139] [Documentation] Operatingsystems linux local mode systemd sc status not working in RHEL 9 (#2984)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

</TabItem>
<TabItem value="Quota" label="Quota">

| Métrique                        | Unité |
|:--------------------------------|:------|
| *quota*#quota.data.usage.bytes  | B     |
| *quota*#quota.files.usage.count | count |

</TabItem>
<TabItem value="Storages" label="Storages">

| Métrique                              | Unité |
|:--------------------------------------|:------|

> | *disk_name*#storage.space.usage.bytes | B     |

Reviewer: ```suggestion
| *disk_name*#storage.space.usage.bytes | B     |
| *partition_name*#storage.space.free.bytes  | B     |
```

| WARNINGUSAGEPRCT  | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                    |                              |             |
| CRITICALUSAGEPRCT | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                    |                              |             |
| EXTRAOPTIONS      | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose --use-new-perfdata |             |

</TabItem>
<TabItem value="Systemd-Journal" label="Systemd-Journal">

| Macro           | Description                                                                                                                                                     | Valeur par défaut | Obligatoire |
|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution. (default: 'cache') | cache             |             |

Reviewer 2: ```suggestion
| SINCE           | Defines the amount of time to look back at messages. Can be minutes (i.e. 5 "minutes ago") or 'cache' to use the timestamp from the last execution. (default: 'cache') | cache             |             |
```

| list-storages [[code](https://github.com/centreon/centreon-plugins/blob/develop/src/os/linux/local/mode/liststorages.pm)]               | Not used in this Monitoring Connector |
| list-systemdservices [[code](https://github.com/centreon/centreon-plugins/blob/develop/src/os/linux/local/mode/listsystemdservices.pm)] | Used for service discovery            |
| load [[code](https://github.com/centreon/centreon-plugins/blob/develop/src/os/linux/local/mode/loadaverage.pm)]                         | OS-Linux-Load-SSH-custom              |

> | lvm [[code](https://github.com/centreon/centreon-plugins/blob/develop/src/os/linux/local/mode/lvm.pm)]                                  | Not used in this Monitoring Connector |

Reviewer 2: LVM?
